medicinskij-academicheskij-jurnal.csl

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Medicinskiy Academicheskiy Joural</title>
```

Comment:
Shouldn't this be "Medicinskiy Akademicheskiy Zhurnal"? (see e.g. the Google results at https://www.google.com/search?q=Medicinskiy+Academicheskiy+Joural)

It's also fine to use Cyrillic.